media-video/mpv/mpv-0.27.1.ebuild

```diff
@@ -0,0 +1,374 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+PYTHON_REQ_USE='threads(+)'
+
+WAF_PV=1.9.8
+
+inherit flag-o-matic gnome2-utils pax-utils python-r1 toolchain-funcs versionator waf-utils xdg-utils
+
+DESCRIPTION="Media player based on MPlayer and mplayer2"
+HOMEPAGE="https://mpv.io/"
+
+if [[ ${PV} != *9999* ]]; then
+	SRC_URI="
+		https://github.com/mpv-player/mpv/archive/v${PV}.tar.gz -> ${P}.tar.gz
+		https://dev.gentoo.org/~kensington/distfiles/${P}-patches-${PR}.tar.xz
+	"
+	KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ppc ~ppc64 ~x86 ~amd64-linux"
+	DOCS=( RELEASE_NOTES )
+else
+	EGIT_REPO_URI="https://github.com/mpv-player/mpv.git"
+	inherit git-r3
+fi
```

Comment:
You're appending to undefined `DOCS` and `SRC_URI` when this branch is used, so please reset them to `DOCS=(); SRC_URI=""` in case something leaked from environment.